### PR TITLE
[Draft] Testing hapigen output for ni-fgen

### PR DESCRIPTION
**Pull request by reckenro**
### What does this Pull Request accomplish?

Using it to test the output from hapigen for fgen. It's in progress in [this PR](https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/357617) in Azure.

### Why should this Pull Request be merged?

It shouldn't. That's why it's a draft request. Just using for diffs and running the build.

### What testing has been done?

Manually checking metadata updates and generated file updates.
System tests still pass.

### CODE REVIEW

File: generated/nifgen/nifgen.proto [hunk 1]
```diff
@@ -159,12 +153,18 @@ enum NiFgenAttribute {
   NIFGEN_ATTRIBUTE_RECORD_COERCIONS = 1050006;
   NIFGEN_ATTRIBUTE_DRIVER_SETUP = 1050007;
   NIFGEN_ATTRIBUTE_INTERCHANGE_CHECK = 1050021;
-  NIFGEN_ATTRIBUTE_CHANNEL_COUNT = 1050203;
```
Comment (reckenro) [inline generated/nifgen/nifgen.proto:162]: Missing attribute

File: generated/nifgen/nifgen.proto [hunk 2]
```diff
@@ -363,25 +374,14 @@ enum NiFgenAttribute {
   NIFGEN_ATTRIBUTE_MAX_SEQUENCE_LENGTH = 1250214;
   NIFGEN_ATTRIBUTE_MAX_LOOP_COUNT = 1250215;
   NIFGEN_ATTRIBUTE_TRIGGER_SOURCE = 1250302;
-  NIFGEN_ATTRIBUTE_BURST_COUNT = 1250350;
```
Comment (reckenro) [inline generated/nifgen/nifgen.proto:366]: Attribute name changed.

File: generated/nifgen/nifgen.proto [hunk 3]
```diff
@@ -363,25 +374,14 @@ enum NiFgenAttribute {
...
+  NIFGEN_ATTRIBUTE_CYCLE_COUNT = 1250350;
 }
 
 enum ByteOrder {
   BYTE_ORDER_NIFGEN_VAL_LITTLE_ENDIAN = 0;
   BYTE_ORDER_NIFGEN_VAL_BIG_ENDIAN = 1;
 }
 
-enum ClockMode {
```
Comment (reckenro) [inline generated/nifgen/nifgen.proto:374]: Missing enum

File: generated/nifgen/nifgen.proto [hunk 4]
```diff
@@ -363,25 +374,14 @@ enum NiFgenAttribute {
...
-  CLOCK_MODE_NIFGEN_VAL_HIGH_RESOLUTION = 0;
-  CLOCK_MODE_NIFGEN_VAL_DIVIDE_DOWN = 1;
-  CLOCK_MODE_NIFGEN_VAL_AUTOMATIC = 2;
-}
-
-enum FrequencyListOptions {
```
Comment (reckenro) [inline generated/nifgen/nifgen.proto:380]: Missing enum

File: generated/nifgen/nifgen.proto (hunk 1 above)
Comment (reckenro) [reply generated/nifgen/nifgen.proto:162]: Now the name's changed

File: generated/nifgen/nifgen.proto [hunk 5]
```diff
@@ -1931,7 +1968,10 @@ message SetAttributeViStringRequest {
   nidevice_grpc.Session vi = 1;
   string channel_name = 2;
   NiFgenAttribute attribute_id = 3;
-  string attribute_value_raw = 4;
+  oneof attribute_value_enum {
```
Comment (reckenro) [inline generated/nifgen/nifgen.proto:1971]: Need to maintain value_raw field number

File: generated/nifgen/nifgen.proto [hunk 6]
```diff
@@ -790,7 +836,10 @@ message CheckAttributeViStringRequest {
   nidevice_grpc.Session vi = 1;
   string channel_name = 2;
   NiFgenAttribute attribute_id = 3;
-  string attribute_value_raw = 4;
```
Comment (reckenro) [inline generated/nifgen/nifgen.proto:793]: Update to keep value_raw field number the same.

**Comment by dmondrik** [comment]
These changes should be rendered obsolete by my PR #754
This PR should be abandoned.

**Pull request closed without merging**